src/cmake-tools.ts

```diff
@@ -1487,17 +1487,16 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
       return;
     }
     await this._setDefaultBuildTarget(target);
-    const drv = await this._cmakeDriver;
-    await this.reRegisterTaskProviderforNewTarget(drv, target);
+    /*const drv = await this._cmakeDriver;
+    this.updateDefaultTargetInTaskProvider(drv, target);*/
   }
 
-  async reRegisterTaskProviderforNewTarget(drv: CMakeDriver | null, target: string | undefined) {
+  /*updateDefaultTargetInTaskProvider(drv: CMakeDriver | null, target: string | undefined) {
```

Comment:
per convention, prefer `target?: string` instead of `target: string | undefined`